Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -610,4 +627,114 @@ namespace AZ::Render
             m_fontDrawInterface->DrawScreenAlignedText3d(m_drawParams, joint->GetName());
         }
     }
+
+    void AtomActorDebugDraw::RenderNodeOrientations(EMotionFX::ActorInstance* actorInstance,
+        AzFramework::DebugDisplayRequests* debugDisplay, float scale)
+    {
+        // Get the actor and the transform data
+        const float unitScale =
+            1.0f / (float)MCore::Distance::ConvertValue(1.0f, MCore::Distance::UNITTYPE_METERS, EMotionFX::GetEMotionFX().GetUnitType());
+        const EMotionFX::Actor* actor = actorInstance->GetActor();
+        const EMotionFX::Skeleton* skeleton = actor->GetSkeleton();
+        const EMotionFX::TransformData* transformData = actorInstance->GetTransformData();
+        const EMotionFX::Pose* pose = transformData->GetCurrentPose();
+        const float constPreScale = scale * unitScale * 3.0f;
+
+        const size_t numEnabled = actorInstance->GetNumEnabledNodes();
+        for (size_t i = 0; i < numEnabled; ++i)
+        {
+            EMotionFX::Node* joint = skeleton->GetNode(actorInstance->GetEnabledNode(i));
+            const size_t jointIndex = joint->GetNodeIndex();
+
+            static const float axisBoneScale = 50.0f;
+            const float size = CalculateBoneScale(actorInstance, joint) * constPreScale * axisBoneScale;
+            AZ::Transform worldTM = pose->GetWorldSpaceTransform(jointIndex).ToAZTransform();
+            RenderLineAxis(debugDisplay, worldTM, size, false /*selected*/);
+        }
+    }
+
+    void AtomActorDebugDraw::RenderLineAxis(
+        AzFramework::DebugDisplayRequests* debugDisplay,
+        AZ::Transform worldTM,
+        float size,
+        bool selected,
+        bool renderAxis,
+        bool renderAxisName)
+    {
+        const float axisHeight = size * 0.7f;
+        const float frontSize = size * 5.0f + 0.2f;
+        const AZ::Vector3 position = worldTM.GetTranslation();
+
+        if (renderAxis)
+        {
+            AZ::Color xSelectedColor;
+            if (selected)
+            {
+                xSelectedColor = AZ::Colors::Orange;
+            }
+            else
+            {
+                xSelectedColor = AZ::Colors::Red;
+            }
```

Comment:
nit (repeated below): This could be simplified with a ternary expression 

```
const AZ::Color xSelectedColor = selected ? AZ::Colors::Orange : AZ::Colors::Red;
```